src/data_management/rasters/RasterSourceTable.tsx

```diff
@@ -25,6 +29,8 @@ export const RasterSourceTable = (props:any) =>  {
   const [deleteFunction, setDeleteFunction] = useState<null | Function>(null);
   const [busyDeleting, setBusyDeleting] = useState<boolean>(false);
   const [showDeleteFailedModal, setShowDeleteFailedModal] = useState<boolean>(false);
+  const scenarioTotalSize = useSelector(getScenarioTotalSize);
```

Comment:
I think you should get `rasterTotalSize` here instead of `scenarioTotalSize`

Reply:
fixed